snuba/migrations/snuba_migrations/outcomes/0001_outcomes.py

```diff
@@ -125,4 +125,7 @@ def backwards_dist(self) -> Sequence[operations.SqlOperation]:
             operations.DropTable(
                 storage_set=StorageSetKey.OUTCOMES, table_name="outcomes_raw_dist",
             ),
+            operations.DropTable(
+                storage_set=StorageSetKey.OUTCOMES, table_name="outcomes_hourly_dist",
+            ),
```

Comment:
I think this needs to replace the drop table statement on lines 122-124, which incorrectly reference the _local table.

Reply:
Ah yes, good catch.